### ISSUE TITLE
Sonoff RF double press on button does not go to learn mode

### ISSUE THREAD

**Issue by gheesung** [arendst/Sonoff-Tasmota#798]
I am using tasmota v5.6.1. The sonoff rf does not go into learn mode when double pressed on the switch.

**Comment by gheesung** [comment]
I have tried multiple consecutive presses, the LED will turn red but not flashing.

Did I setup the firmware wrongly?

**Comment by arendst** [comment]
RF is NOT controlled by Tasmota. If the LED turns red that means that the RF microcontroller is waiting for your RF command. If it cannot recognize your RF remote control it won't flash.

**Comment by gheesung** [comment]
I have discovered the problem. I am trying to power the board using the FTDI adapter and it does not have enough current to power both ESP and RF board. 

In the wiki page, the instruction to prepare the Sonoff RF is to solder a jumper across R21. This is not necessary as I also found out that I can just short the inner pin of R21 when powering up the device. It will go into programming mode.

![Alternative flashing](https://4.bp.blogspot.com/-r5PH3spM4PQ/WaRTEFVBYZI/AAAAAAAA0lk/cy6OrdcexZkJd9GGQlsIpQRPTuUqPIgMACLcBGAs/s320/Clipboard01.png)

![Alternative flashing](https://4.bp.blogspot.com/-YCf1RDLuY3U/WaRVa-Z8SkI/AAAAAAAA0lw/NauNUIjnVRwC4Mp2-Ig3_4IPPMB7OBqTwCLcBGAs/s320/Clipboard01.png)
